src/main/java/org/skriptlang/skript/bukkit/spawner/elements/expressions/spawner/ExprSpawnCount.java

```diff
@@ -0,0 +1,81 @@
+package org.skriptlang.skript.bukkit.spawner.elements.expressions.spawner;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.spawner.Spawner;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.spawner.SpawnerModule;
+import org.skriptlang.skript.bukkit.spawner.util.SpawnerUtils;
+
+@Name("Spawner - Spawn Count")
+@Description({
+	"Returns the spawn count. The spawn count is the number of entities "
+		+ "that the spawner will attempt to spawn each spawn attempt. By default, the value is 4.",
+	"If the spawner entity is an item, the spawn count is the number of stacks of items to spawn.",
+	"",
+	"Spawners are creature spawners and spawner minecarts."
+})
+@Examples({
+	"set spawn count of target block to 5",
+	"add 2 to spawn count of target block",
+	"remove 1 from spawn count of target block",
+	"reset spawn count of target block"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.21+")
+public class ExprSpawnCount extends SimplePropertyExpression<Object, Integer> {
+
+	static {
+		register(SpawnerModule.SYNTAX_REGISTRY, ExprSpawnCount.class, Integer.class, "spawn count", "entities/blocks");
```

Comment:
Same as before, should include `spawner`. More so here than others since `spawn count` sounds extremely universal, and could be confusing if we think of server spawn counts